tests/storage/restore.go

```diff
@@ -229,6 +229,10 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 					vm, err = virtClient.VirtualMachine(vm.Namespace).Get(vm.Name, &metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
 
+					if vm.Status.SnapshotInProgress != nil {
```

Comment:
I think the PR I did the was merged yesterday fixes this: https://github.com/kubevirt/kubevirt/pull/6146

Reply:
ah right don't think this hurts anything though